examples/IRB140/irb_140_manual_transmissions.urdf

```diff
@@ -0,0 +1,361 @@
+<robot

```

Comment:
i think we should probably keep just one urdf hanging around.  we don't want to start having copies w/ and w/o transmission elements, etc.